docs/source/fitting/fitfunctions/ChudleyElliotSQE.rst

```diff
@@ -0,0 +1,48 @@
+.. _func-Chudley-ElliotSQE:
+
+================
+ChudleyElliotSQE
+================
+
+.. index:: ChudleyElliotSQE
+
+Description
+-----------
+
+This fitting function models the dynamic structure factor
+for a particle undergoing jump diffusion using the Chudley-Elliot Jump diffusion model [1]_.
+
+.. math::
+
+   S(Q,E) = Height \cdot \frac{1}{\pi} \frac{\Gamma}{\Gamma^2+(E-Centre)^2}
+
+   \Gamma = \frac{\hbar}{\tau} \cdot \left(1 - \frac{sin(Ql)}{Ql}\right)
```

Comment:
Am I right that based on Eq. 25 of Ref. 1 the authors of the reference use a slightly different fit function than the one you define here? If I am not mistaken, they have a factor 2*exp(-2*W) instead of Height/Pi. If that is the case, is there a benefit in using your parametrisation for S(Q, E) instead of theirs or is there a physical reason that makes your parametrisation more feasible? Or am I misunderstanding something?

Reply:
The parameter W in the exponential is another function involving Q. It can be measured in a different experiment. Most users have no need of it so our fitting function is kept simple.

Reply:
Got it, thanks for the explanation